src/main/java/com/philips/swcoe/cerberus/cerebellum/codemetrics/java/CodeMetricsHorizontalWriterService.java

```diff
@@ -15,9 +15,9 @@
 public class CodeMetricsHorizontalWriterService extends AbstractCodeMetricsWriterService  {
```

Comment:
See comment earlier on name of AbstractCodeMetricsWriterService. If you clean this naming noise then here you can rename also this class to HorizonalMetricsWriter. So we have a nice flow for the reader to understand, i.e., "This is a kind of metrics writer for my code metrics application", "What kind of metrics writer?", "A horizontal metrics writer". Simple and aligned with how you would explain the relationship verbally of you were talking someone through the code to explain.

Reply:
Yes, it is now corrected